src/renderer/tvgPaint.cpp

```diff
@@ -73,11 +73,14 @@ static Result _clipRect(RenderMethod* renderer, const Point* pts, const Matrix&
 }
 
 
-static Result _compFastTrack(RenderMethod* renderer, Paint* cmpTarget, const Matrix& pm, RenderRegion& before)
+static Result _compFastTrack(RenderMethod* renderer, Paint* cmpTarget, const Matrix& pm, RenderRegion& before, bool clip = false)
 {
     /* Access Shape class by Paint is bad... but it's ok still it's an internal usage. */
     auto shape = static_cast<Shape*>(cmpTarget);
 
+    //Clipping with a stroke prevents fast track
+    if (clip && SHAPE(shape)->rs.strokeWidth() > 0.0f) return Result::InsufficientCondition;
```

Comment:
Please add the condition around 256 line. Thanks.

Reply:
@hermet changed, but there was also another issue present even if the stroke width was 0. I pushed it as a separate commit in this pr and edited description in pr so that there are samples to test each commit.